app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -402,7 +404,7 @@ class ProfileChooserFragmentTest {
       testCoroutineDispatchers.runCurrent()
       onView(withId(R.id.administrator_controls_linear_layout)).perform(click())
       intended(hasComponent(AdminAuthActivity::class.java.name))
-      intended(hasExtra(AdminAuthActivity.getIntentKey(), 0))
+      intended(hasExtra(AdminAuthActivity.getIntentKey(), AdminAuthEnum.PROFILE_ADMIN_CONTROLS.value))
```

Comment:
aha, This looks more readable now. Thanks

Reply:
Just to confirm, is all tests are passing now on both espresso and robolectric, other than the espresso issue you had mentioned?

Reply:
As all tests are passing now, please update the screenshots.

Reply:
Done

Reply:
> Just to confirm, is all tests are passing now on both espresso and robolectric, other than the espresso issue you had mentioned?

Yes all test cases are passing now, no issues